### PR TITLE
[WIP] Added Let's Do Something post.

### PR DESCRIPTION
**Pull request by ashfurrow**
Trying to be positive here while also telling a story about why it's important. 

I've asked for feedback on this from several people directly. 

If you have suggestions, you're welcome to comment here or send an email to ash@ashfurrow.com directly to provide anonymous feedback. 

BTW, the latest is deployed to staging: http://staging.ashfurrow.com/blog/let-s-do-something/

### CODE REVIEW

**Comment by ashfurrow** [comment]
Re-reading this after this morning, and I feel the intro drags on. I don't like the following at all:

> It’s pretty negative, I think, to get angry at people on a list (who didn’t ask to be there) because the constituency of the list isn’t diverse.

Like, why I might feel bad doesn't really matter. Talking about how them complaining about people on the list doesn't matter in the context of helping.

File: source/blog/2015-06-15-let-s-do-something.html.markdown [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+Well, that's kind of a cop out, isn't it? 
+
+I mean, think about it. You're in tech and you're a not a white cishet dude. Your life and your career are already more difficult than mine are, but now on top of that, some white dudes expect you to singlehandedly try to fix the diversity problem in tech _and then_ they'll signal boost you. 
+
+It must be exhausting. To experience all the problems of the tech world and then _also_ be expected to be a role model to every other member of ever underrepresented group who might someday get into tech. 
+
+That's a lot of responsibility. 
+
+I think it's important to remind ourselves that some people just want to do their jobs, and that should be OK. It should be OK to just wants to make apps without having extra responsibilities to try and fix tech's problems. Responsibilities no one really places on white cishet men.
```
Comment (segiddins) [inline source/blog/2015-06-15-let-s-do-something.html.markdown]: `s/wants/want/`

Comment (ashfurrow) [inline source/blog/2015-06-15-let-s-do-something.html.markdown]: Thanks!

**Comment by samdmarshall** [comment]
When this list originally went up I was pretty critical of it due to the lack of diversity. My opinions of the make-up of this list hasn’t changed but I don’t think that it is surprising. Tech, and more specifically the visible iOS/OS X community is made up of cis white dudes. Part of this is born from the fact that the apple community has been a majority of cis white dudes because you have to be fairly privileged to work with these products and technologies. 

The key wording i used there was “visible”, being visible is extremely taxing regardless of who you are. I go the extra mile to be visible when I really don’t have to and most times isn’t worth trying to do. I do it because it can make things easier for other marginalized people. There are a multitude of reasons for that but much of it comes down: the act of being visible means that someone else that is less able to handle doing that doesn’t have to be visible, and being visible means others feel less alone if they try to be visible.

Part of the problem here is that the culture our industry has is really attracted to the idea of putting people on pedestals. So many things we do is based on who we idolize:
* libraries we use
* dependency managers we use
* syntax styles
* podcasts
* code architecture

It is rather ridiculous. As a whole I have a problem with doing this because it sends the wrong type of message, especially about diversity. To be honest I get email not infrequently that can be paraphrased as “you are really amazing, what you do is really cool, how could i do …” and that makes me really uncomfortable. I don’t like being seen as a figure of authority when I am still very much still learning things.

If you are going to make a list by saying “hey here are a bunch of cool people that I like” that is neat and I’ll probably look at that list. The way that differs from the “52 people” list is that iOS already has a clubhouse problem. There is nothing wrong with that list being made, and I think the vast majority of developers would learn a lot from following those 52 people. My original comments were directed at the fact of how homogenous it is and that is disappointing.

I cannot speak for anyone but myself and my own experiences; what I have seen is that overall the iOS community does have more diversity than we might think it does from a sampling of speaker line-ups or photos of crowds at WWDC. What I have seen and been told by other marginalized people is that most of us do not want to be on lists because we also happen to not be cishet white men. Due to that, I find many lists that aim to highlight marginalized people to be distasteful. It turns the fact that I am queer/trans/non-binary into a political statement rather than a description of who i am. I am not an activist, I don’t use those words as a statement beyond that they are meaningful to me. Having someone take them and turn around to use them to say “here is representation” is really wrong to me. I would say that others i’ve spoken with would also feel similarly about it. Because these reasons, it also makes being on such lists potentially dangerous. Many people don’t have the luxury of being able to have the emotional or time to invest with the rigors of being visible. Many people could have themselves put in jeopardy if they were seen to be “political” about who they are in their place of employment or even their lives. 

There is a need for better representation in tech, and i know we can do better in the iOS community. As mentioned I think part of the problem is who we immediately turn to for help or advice. As an example, I follow 4 time the number of people on the “52 people” list. Men are the minority on my list. The vast majority of them also spend their time doing their jobs. Nobody is going to make a list lauding the fact that they do their jobs. If someone did make such a list, I feel it would quickly become a list for the sake of “diversity”.

How can we have a more diverse community if we don’t have people actively making it more diverse? We can start by normalizing the fact that there are people here already. Many people aren’t looking for heroes, only to feel like who they are is normal in the environment that they work in. Signal boosting people or otherwise normalizing the inclusion of other people in conversations both achieve that goal. Otherwise we are perpetuating the same experience of a bunch of dudes standing around talking, be that at a conference party or on twitter. 

I have a bunch more things I could say about specific details i’ve mentioned but I think this gets the majority of the points I wanted to make across. I personally have issues with how we use lists and by extension then treat people. This isn’t a pipeline problem, and even if it was it — the only thing this will do is push the burden off of addressing how people are impacted by the lack of inclusion right now. 

To summarize: I don’t know if there is an easy way to do this without normalizing that there are people more than cishet white men, and then work to not be shits to each other so these people can feel welcome in our community. We don’t want to become token representation members of the community. I feel that the safest way for people to do normalize this is to signal boost and not feel like this lack of representation is an action-item or otherwise treat this as some engineering problem to solve. It is terribly easy for us to design solutions to this diversity problem when it isn’t on a deadline — not unlike how we treat our coding problems.

**Comment by ashfurrow** [comment]
Thanks for the thoughtful feedback, Sam. There's a lot there, and I appreciate the time you took to write it up.

**Comment by ashfurrow** [comment]
So, after thinking this over, it's not really appropriate of me to try and spearhead this. I want to help, but this isn't helpful. 

Thanks to everyone, especially @samdmarshall, for the feedback. I have a broader perspective than I did before. Thank you for that.
